src/test/java/com/debatetimer/domain/MemberTest.java

```diff
@@ -0,0 +1,30 @@
+package com.debatetimer.domain;
+
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.ValueSource;
+
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+class MemberTest {
+
+    @Nested
+    class Validate {
+
+        @ParameterizedTest
+        @ValueSource(ints = {0, 11})
+        void 닉네임은_1자_이상_20자_이하여야_한다(int length) {
```

Comment:
10자 이하여야 한다는 점에서 테스트명 수정이 필요해보입니다! 추가적으로 @NullAndEmptySource를 같이 붙여주면 empty  및 null 검증도 함께 할 수 있을 것 같습니당

```suggestion
        @ParameterizedTest
        @NullAndEmptySource
        @ValueSource(ints = {0, 11})
        void 닉네임은_1자_이상_10자_이하여야_한다(int length) {
```